### PR TITLE
Add generate yt object

### PR DESCRIPTION
**Pull request by HerbertIV**

### CODE REVIEW

File: src/Services/VideoService.php [hunk 1]
```diff
@@ -71,13 +57,6 @@ public function relatedToVideoId($part, $params) {
...
-
-		} catch (\Google_Service_Exception $e) {
-			throw new Exception($e->getMessage(), 1);
-
-		} catch (\Google_Exception $e) {
-			throw new Exception($e->getMessage(), 1);
-
 		} catch (Exception $e) {
 			throw new Exception($e->getMessage(), 1);
 		}
```
Comment (KrzysztofDziedziechEscolasoft) [inline src/Services/VideoService.php]: why you make try {} catch {} if you catch all exceptions and throw Exception ?

File: src/Services/YoutubeService.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    public function __construct(
+        AuthenticateServiceContract $authenticateServiceContract,
+        LiveStreamServiceContract $liveStreamServiceContract
+    ) {
+        $this->authenticateServiceContract = $authenticateServiceContract;
+        $this->liveStreamServiceContract = $liveStreamServiceContract;
+    }
+
+    public function generateYTStream(YTBroadcastDto $YTBroadcastDto): ?YTLiveDto
```
Comment (KrzysztofDziedziechEscolasoft) [inline src/Services/YoutubeService.php:25]:
```suggestion
    public function generateYTStream(YTBroadcastDto $yTBroadcastDto): ?YTLiveDto
```

File: src/Services/YoutubeService.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        $token = $this->authenticateServiceContract->refreshToken(config('youtube.refresh_token'));
+        return $this->liveStreamServiceContract->broadcast($token, $YTBroadcastDto);
+    }
+
+    public function setRefreshToken($code)
```
Comment (KrzysztofDziedziechEscolasoft) [inline src/Services/YoutubeService.php:31]:
```suggestion
    public function setRefreshToken($code): void
```

**Review by KrzysztofDziedziechEscolasoft — APPROVED** [review]
fix tests

File: database/migrations/2022_03_04_152905_add_ccolumns_y_t_for_webinar_table.php [hunk 4]
```diff
@@ -15,6 +15,7 @@ public function up()
     {
         if (Schema::hasTable('webinars')) {
             Schema::table('webinars', function (Blueprint $table) {
+                $table->string('yt_id', 30)->nullable();
                 $table->string('yt_url', 255)->nullable();
                 $table->string('yt_stream_url', 255)->nullable();
                 $table->string('yt_stream_key', 255)->nullable();
```
Comment (qunabu) [inline database/migrations/2022_03_04_152905_add_ccolumns_y_t_for_webinar_table.php]: imho this should be in `webinars` package not here

**Review by qunabu — CHANGES_REQUESTED** [review]

**Review by KrzysztofDziedziechEscolasoft — APPROVED** [review]

**Review by qunabu — APPROVED** [review]

**Pull request merged**